maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRunGoalsPhase.java

```diff
@@ -71,11 +72,12 @@ protected ReleaseResult execute( ReleaseDescriptor releaseDescriptor, ReleaseEnv
             String goals = getGoals( releaseDescriptor );
             if ( !StringUtils.isEmpty( goals ) )
             {
-                logInfo( result, "Executing goals '" + goals + "'..." );
-                if ( logArguments && ( additionalArguments != null ) )
+                logInfo( result, "Executing goals '" + buffer().strong( goals ) + "'..." );
+                if ( logArguments )
                 {
                     // logging arguments may log secrets: should be activated only on dryRun
-                    logInfo( result, "    with additional arguments: " + additionalArguments );
+                    logInfo( result, "    with additional arguments: "
+                        + ( additionalArguments == null ? "" : additionalArguments ) );
```

Comment:
The purpose is to show whether there are addArgs or not? Consistency?

Reply:
yes, not seeing that additional arguments were taken into account but were empty was confusing

Reply:
Maybe this is just a matter of taste, but was about: `(none)`

Reply:
you're right